src/Core/Content/Product/Aggregate/ProductPrice/ProductPriceCollection.php

```diff
@@ -34,6 +34,13 @@ public function sortByQuantity(): void
         });
     }
 
+    public function sortByPrice(): void
+    {
+        $this->sort(function (ProductPriceEntity $a, ProductPriceEntity $b) {
+            return $a->getPrice()->first()->getGross() <=> $b->getPrice()->first()->getGross();
```

Comment:
Please provide a unit test for this. Either add a new `ProductPriceCollectionTest` or cover this method in the existing `Shopware\Core\Content\Test\Product\Repository\ProductRepositoryTest`